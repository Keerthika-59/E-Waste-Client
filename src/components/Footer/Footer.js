import React from 'react';
import './Footer.css';
import { Link } from 'react-router-dom';

function Footer() {
  return (
    <div className='footer-container' style={{ marginTop: "15%"}}>
      <section className='footer-subscription'>
        <p className='footer-subscription-heading'>
          Join us for a better environment
        </p>
        <p className='footer-subscription-text'>
          You can unsubscribe at any time.
        </p>        
      </section>
      <div className='footer-links'>
        <div className='footer-link-wrapper'>
          <div className='footer-link-items'>
            <h2>About Us</h2>
            <Link className="footerlinks" to='/sign-up'>How it works</Link>
            <Link className="footerlinks" to='/'>Testimonials</Link>
            <Link className="footerlinks" to='/'>Careers</Link>
            <Link className="footerlinks" to='/'>Terms of Service</Link>
          </div>
          <div className='footer-link-items'>
            <h2>Contact Us</h2>
            <Link className="footerlinks"  to='/ContactUs'>Contact</Link>
            <Link className="footerlinks"  to='/'>Support</Link>
            <Link className="footerlinks"  to='/'>Destinations</Link>
            <Link className="footerlinks"  to='/'>Sponsorships</Link>
          </div>
        </div>
        <div className='footer-link-wrapper'>
         </div>
      </div>
      <section className='social-media'>
        <div className='social-media-wrap'>
          <div className='footer-logo'>
           <hr></hr>
          </div>
          {/* <small class='website-rights'>TRVL Â© 2020</small> */}
          <div className='socialicons'>
            <Link
              className='social-icon-link facebook'
              to=''
              target='_blank'
              aria-label='Facebook'
            >
              <i className='fab fa-facebook-f' href="https://www.facebook.com/"/>
            </Link>
            <Link
              className='social-icon-link instagram'
              to='/'
              target='_blank'
              aria-label='Instagram'
            >
              <i className='fab fa-instagram' href="https://www.facebook.com/" />
            </Link>
            <Link
              className='social-icon-link youtube'
              to='/'
              target='_blank'
              aria-label='Youtube'
            >
              <i className='fab fa-youtube' href="https://www.facebook.com/" />
            </Link>
            <Link
              className='social-icon-link twitter'
              to='/'
              target='_blank'
              aria-label='Twitter'
            >
              <i className='fab fa-twitter' href="https://www.facebook.com/" />
            </Link>
            <Link
              className='social-icon-link twitter'
              to='/'
              target='_blank'
              aria-label='LinkedIn'
            >
              <i className='fab fa-linkedin' href="https://www.facebook.com/" />
            </Link>
          </div>
        </div>
      </section>
    </div>
  );
}
export default Footer;
