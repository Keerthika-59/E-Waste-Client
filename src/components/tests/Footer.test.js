import React from "react";
import { shallow } from 'enzyme';
import ReactDOM from "react-dom";
import { Route } from "react-router-dom";
import Footer from "../Footer/Footer";

test("renders footer", () => {
    const div = document.createElement("div");
    <Route>
      ReactDom.render(
      <Footer />
      ,div);
    </Route>;
    ReactDOM.unmountComponentAtNode(div);
  });
  test('should test Footer component', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
   });