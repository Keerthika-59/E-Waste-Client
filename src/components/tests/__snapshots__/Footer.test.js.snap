// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Footer component 1`] = `
<div
  className="footer-container"
>
  <section
    className="footer-subscription"
  >
    <p
      className="footer-subscription-heading"
    >
      Join us for a better environment
    </p>
  </section>
  <div
    className="footer-links"
  >
    <div
      className="footer-link-wrapper"
    >
      <div
        className="footer-link-items"
      >
        <h2>
          About Us
        </h2>
        <Link
          className="footerlinks"
          to="/AboutUs"
        >
          About Us
        </Link>
        <Link
          className="footerlinks"
          to="/Goals"
        >
          Goals
        </Link>
        <Link
          className="footerlinks"
          to="/Achievements"
        >
          Achievements
        </Link>
        <Link
          className="footerlinks"
          to="/OurTeam"
        >
          Our Team
        </Link>
      </div>
      <div
        className="footer-link-items"
      >
        <h2>
          Services
        </h2>
        <Link
          className="footerlinks"
          to="/Services"
        >
          Our Services
        </Link>
        <Link
          className="footerlinks"
          to="/Ewaste"
        >
          Ewaste
        </Link>
        <Link
          className="footerlinks"
          to="/Disposing"
        >
          Dispose
        </Link>
        <Link
          className="footerlinks"
          to="/Donating"
        >
          Donate
        </Link>
      </div>
      <div
        className="footer-link-items"
      >
        <h2>
          Contact Us
        </h2>
        <Link
          className="footerlinks"
          to="/ContactUs"
        >
          Contact
        </Link>
        <Link
          className="footerlinks"
          to="/Testimonials"
        >
          Testimonials
        </Link>
        <Link
          className="footerlinks"
          to="/RepresentativeSignUp"
        >
          Join us
        </Link>
      </div>
    </div>
    <div
      className="footer-link-wrapper"
    />
  </div>
  <section
    className="social-media"
  >
    <div
      className="social-media-wrap"
    >
      <div
        className="footer-logo"
      >
        <hr />
      </div>
      <div
        className="socialicons"
      >
        <Link
          aria-label="Facebook"
          className="social-icon-link facebook"
          target="_blank"
          to=""
        >
          <a
            href="https://www.facebook.com/"
          >
            <i
              className="fab fa-facebook-f"
              href="https://www.facebook.com/"
            />
          </a>
        </Link>
        <Link
          aria-label="Instagram"
          className="social-icon-link instagram"
          target="_blank"
          to="/"
        >
          <a
            href="https://www.instagram.com/"
          >
            <i
              className="fab fa-instagram"
              href="https://www.facebook.com/"
            />
          </a>
        </Link>
        <Link
          aria-label="Youtube"
          className="social-icon-link youtube"
          target="_blank"
          to="/"
        >
          <a
            href="https://www.youtube.com/?gl=IN"
          >
             
            <i
              className="fab fa-youtube"
              href="https://www.facebook.com/"
            />
          </a>
        </Link>
        <Link
          aria-label="Twitter"
          className="social-icon-link twitter"
          target="_blank"
          to="/"
        >
          <a
            href="https://twitter.com/?lang=en-in"
          >
             
            <i
              className="fab fa-twitter"
              href="https://www.facebook.com/"
            />
          </a>
        </Link>
        <Link
          aria-label="LinkedIn"
          className="social-icon-link twitter"
          target="_blank"
          to="/"
        >
          <a
            href="https://www.linkedin.com/login/"
          >
            <i
              className="fab fa-linkedin"
              href="https://www.facebook.com/"
            />
          </a>
        </Link>
      </div>
    </div>
  </section>
</div>
`;
